ksql-engine/src/main/java/io/confluent/ksql/function/FunctionRegistry.java

```diff
@@ -151,6 +153,12 @@ private void init() {
         "EXTRACTJSONFIELD", JsonExtractStringKudf.class);
     addFunction(getStringFromJson);
 
+    Schema array = SchemaBuilder.array(Schema.STRING_SCHEMA).build();
+    KsqlFunction jsonArrayContainsString = new KsqlFunction(
+            Schema.BOOLEAN_SCHEMA, Arrays.asList(array, Schema.STRING_SCHEMA),
+            "JSON_ARRAY_CONTAINS", JsonArrayContainsKudf.class);
+    addFunction(jsonArrayContainsString);
```

Comment:
I have an issue registering my function passing arguments to it. I'm expecting that 'evaluate' method passes json string as first argment of the array, and the second argument is search value.

However, when I execute fullowing query: `select JSON_ARRAY_CONAINS(["abc", "aaa", "dd"], 'aaa') from whatever`.  evaluate function gets the string array of ["abc", "aaa", "dd"].